app/models/setting.rb

```diff
@@ -0,0 +1,17 @@
+# RailsSettings Model
+class Setting < RailsSettings::Base
+  cache_prefix { "v1" }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Prefer single-quoted strings when you don't need string interpolation or special symbols.](https://app.codacy.com/manual/VoxPupuli/vox-pupuli-tasks/pullRequest?prid=5443028)